src/api/feedback.api.js

```diff
@@ -6,19 +6,19 @@ import { API_SERVICES_BASE_URL, APP_VERSION } from '@/config'
 import log from '@/utils/logging'
 
 /**
+ * @param {String} subject Mandatory
  * @param {String} text Mandatory
- * @param {Number} rating Optional
- * @param {Number} maxRating Optional
- * @param {String} kmlFileUrl Optional
- * @param {String} email Optional
+ * @param {String | null} [kmlFileUrl=null] Default is `null`
+ * @param {String | null} [email=null] Default is `null`
+ * @param {File | null} [attachment=null] Default is `null`
  * @returns {Promise<Boolean>} True if successful, false otherwise
  */
 export default async function sendFeedback(
+    subject,
     text,
-    rating = null,
-    maxRating = null,
     kmlFileUrl = null,
-    email = null
+    email = null,
+    attachment = null
```

Comment:
When already changing this, it would be better to use an options object
`(subject, text, {email = null, kmlFileUrl = null, attachement = null})`
So below you don't have to pass optional argument and if you want to pass only one you can simply do `sendFeedback(subject, text, {email: 'hello@world.com'})`